nylas/api/wsgi.py

```diff
@@ -53,6 +53,14 @@ def log_request(self):
         if request_uid is not None:
             additional_context['request_uid'] = request_uid
 
+        # pywsgi negates the status code if there was a socket error
+        # To make the logs clearer, use the positive status code and include
+        # the socket error
+        if status < 0:
+            additional_context['error'] = 'socket.error'
+            additional_context['error_message'] = getattr(self, 'status', None)
+            status = -status
```

Comment:
I think this would be clearer as `status = abs(status)` but I don't think it's a huge deal so if you prefer this way feel free to keep it :)